prometheus-exporters/openstack-exporter/alerts/cinder.alerts

```diff
@@ -24,8 +24,8 @@ groups:
       service: cinder
       playbook: docs/support/playbook/cinder/cinder-low-free-space.html
     annotations:
-      description: 'Cinder backend {{ $labels.shard }}/{{ $labels.backend }} has 3 or less datastores with > 30% free space left'
-      summary: 'Cinder backend {{ $labels.shard }}/{{ $labels.backend }} has 3 or less3 datastores with > 30% free space left'
+      description: 'Cinder backend {{ $labels.shard }}/{{ $labels.backend }} has 2 or less datastores with > 30% free space left'
+      summary: 'Cinder backend {{ $labels.shard }}/{{ $labels.backend }} has 2 or less datastores with > 30% free space left'
```

Comment:
Doesn't the expression say "< 2" and thus it cannot be 2 while the alert fires? -> "has less than 2 datastores"

Reply:
done